### ISSUE TITLE
Camera reconnection

### ISSUE THREAD

**Issue by alexandruradovici** [alliedvision/VimbaPython#106]
I am trying to disconnect and reconnect to a Alvium 1800 U-1620 USB camera. It seems that the camera returns error -12 unless I restart the application. I have attached a video showing the behavior in VimbaViewer. I am using Ubuntu 20.04 LTS.

https://user-images.githubusercontent.com/2838177/165515456-96ad3bda-1f87-405e-ac31-af58e8206e1f.mp4

**Comment by sahandhsn** [comment]
yes this is my problem too. Sometimes though closing the program does not solve it and I have to disconnect the USB Cable and connect it again for it to work. 
Same issue in VimbaPython, if I connect the USB cable and run the example code asynchronous_grab.py. I get the frames. but if I run it again I get the error: 
`vimba.error.VimbaTimeout: Frame capturing on Camera 'DEV_XXXXXXXXXXXX' timed out.
`
I am using Ubuntu 20.04 in a virtual machine.

**Comment by arunprakash-avt** [comment]
Hi,
The above issue is because of the Virtual Machine because the VM takes more CPU performance and the USB drivers cannot communicate to the PC and causes the camera timeout. I request you to test it on a Ubuntu system running not from Virtual machine.

Thanks
Arun Prakash

**Comment by Lineup12** [comment]
@arunprakash-avt 
Thanks a lot, yes using the camera without a virtual machine solved the problem.
